### ISSUE TITLE
感谢您的工作，想请教一下您的模型怎么得到的81.8的过程可以吗？

### ISSUE THREAD

**Issue by yhl2018** [zcablii/LSKNet#32]
### Model/Dataset/Scheduler description

我看到您训练的config是拿训练和评估集作为训练，评估集作为val，训练了12个epoch，在您readme中LSKNet_S* log评估集中的结果为mAP: 0.8691，上传到dota上得到的结果是81.85.我想请教的是，这个val的结果和test集上的结果有相关性吗？您提交的test集结果是怎么分解的test，做了什么特殊处理吗？

### Open source status

- [ ] The model implementation is available
- [ ] The model weights are available.

**Comment by zcablii** [comment]
We utilize both the training and validation sets combined for a total of 12 epochs. The validation performed on the validation set ensures the training process proceeds smoothly, detecting any potential issues like NaN or model collapse.

Given that the validation data is exposed to the model during training, it's expected that the mAP on the validation set would be higher compared to the test set. This discrepancy is a natural consequence of the model's familiarity with the validation data due to exposure during training.
